mailchimp-marketing/1.0/modules/ROOT/pages/mailchimp-marketing-connector-reference.adoc

```diff
@@ -0,0 +1,1412 @@
+= Mailchimp Marketing Connector 1.0 Reference
+
+
+
+== Configurations
+---
+[[Config]]
+=== Config
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<Config_ApiKey, Api Key Connection Provider>>
+* <<Config_OauthAuthorizationCode, OAuth Authorization Code Connection Provider>>
+ | Connection types for this configuration. | | x
+| Response Timeout a| Number |  How long Mule waits for a response to complete before the request times out. |  60 |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Response Timeout* field. |  SECONDS |
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER | How the connector streams request content to the remote service:
+
+* `AUTO` (default)
++
+Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`
++
+Connector always streams the request content.
+
```

Comment:
Most of the rendered list items show "+" in the text. It looks like the auto-generation process left out "+" in some places so the lists were rendered incorrectly.